packages/db/src/search/field-mapping.ts

```diff
@@ -145,15 +151,50 @@ const collectionMapping : { [key: string]: Function } = {
 };
 
 const executionMapping : { [key: string]: Function } = {
+  asyncOperationId: (value?: string) => ({
+    asyncOperationId: value,
+  }),
+  asyncOperationCumulusId: (value?: string) => ({
+    async_operation_cumulus_id: value,
+  }),
   arn: (value?: string) => ({
     arn: value,
   }),
   createdAt: (value?: string) => ({
     created_at: value && new Date(Number(value)),
   }),
+  duration: (value?: string) => ({
+    duration: value && Number(value),
+  }),
+  //need to take another look at this and originalPayload
+  finalPayload: (value?: string) => ({
```

Comment:
I think finalPayload and originalPayload are object in schema, and not queryable.